### ISSUE TITLE
Coordinate: Create an servicecanada.gc.ca inbox for a11y feedback

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#2593]
The second channel for receiving a11y feedback is through email. 

- [x] An inbox that will be monitored by someone.
- [x] Find out who the someone is. Is it the App team, the Support team: will be PM
- [x] Decide on what words we want in the email address
- [x] Complete form fields in [this doc](https://docs.google.com/document/d/1FBZZzgDrga-RXZM24oMFVeLkc3WMkH7oVpcd9GlwVIY/edit?tab=t.0)

**Comment by schancase** [comment]
adding sprint goal tag because this is parented by a sprint goal feature

**Comment by amazingphilippe** [comment]
There is a process for this. This is called a general delivery mailbox. We need to submit a request, and it might be approved. I can't find out what information is needed to complete the request. It must be done by @AndreanneTrudeau, our Director/Manager. Team leaders are not allowed either.

**Comment by YedidaZalik** [comment]
Yesterday Carolyn said PM will be person monitoring inbox.

Gathered docs for making mailbox request from Phil. 

Checking with Phil and Carolyn about bilingual email address.

**Comment by YedidaZalik** [comment]
We decided on  accessible+notification@servicecanada.gc.ca  as it's both bilingual and brief
There’s more detail on how we decided in this [slack thread.](https://gcdigital.slack.com/archives/C06KQFM10BU/p1764791130031769)
I cannot fill out the form for Andréanne to sign as I don't have authority to access it
I'm gathering the info she might need and checking with Carolyn and Phil on some of that

**Comment by YedidaZalik** [comment]
Confirmed answers, Melissa reviewed form, I shared with Andréanne, who will update us today or tomorrow

**Comment by YedidaZalik** [comment]
Andréanne submitted form so we're waiting for reply

**Comment by schancase** [comment]
blocked for now - we are dependent on national desk support now

**Comment by YedidaZalik** [comment]
National desk team came back with a clarification question:
J’aurais besoin version française – anglaise des acronymes pour la création de votre boite GD  
ex : EDSC.ACCESSIBILITE.NOTIFICATION-ACCESSIBLE.NOTIFICATION.ESDC 
display name : ACCESSIBILITE NOTIFICATION / ACCESSIBLE NOTIFICATION (EDSC/ESDC)

Phil and I just discussed and requested Andréanne to reply with the following questions:

(1) We chose “ACCESSIBLE” because it works in both languages. We would prefer to use just this one word so the email address is short rather than adding “ACCESSIBILITE." Since it's bilingual, could we have “ACCESSIBLE” only?

(2) Again with the hope of keeping it brief: Are we required to include the acronyms EDSC/ESDC since we have already included  bilingual word for our product (NOTIFICATION)?

**Comment by YedidaZalik** [comment]
Andréanne got this reply
Bonjour,
La seule chose que je pourrais faire est ceci
EDSC.ACCESSIBLE-NOTIFICATION.ESDC
Display name: ACCESSIBLE / NOTIFICATION (EDSC/ESDC)

Phil and I are concerned about ESDC in the name because our users are familiar with GC NOTIFY not ESDC NOTIFY.
So we would prefer the following
Display name: NOTIFICATION GC ACCESSIBLE GC NOTIFY
Email address: NOTIFICATION.GC.ACCESSIBLE.GC.NOTIFY
If including ESDC is mandatory, then we could use the following display name:
NOTIFICATION GC ACCESSIBLE GC NOTIFY (EDSC|ESDC)

**Comment by schancase** [comment]
moving to in progress so that we can respond to NSD. Might also be good to ask Jimmy P.

**Comment by YedidaZalik** [comment]
ESDC assigned the following

Account name: EDSC.NOTIFICATION.GC-ACCESSIBLE.GC.NOTIFY.ESDC@servicecanada.gc.ca

Display name: NOTIFICATION GC / ACCESSIBLE GC.NOTIFY (EDSC/ESDC)

We're requesting they change display name to one of the following:

NOTIFICATION GC / ACCESSIBLE / GC NOTIFY (EDSC/ESDC)

NOTIFICATION GC ACCESSIBLE / ACCESSIBLE GC NOTIFY (EDSC/ESDC)

**Comment by YedidaZalik** [comment]
Andréanne received confirmation from National Service Desk that we can use NOTIFICATION GC / ACCESSIBLE / GC NOTIFY (EDSC/ESDC)
I've updated the designs to show
Account name: EDSC.NOTIFICATION.GC-ACCESSIBLE.GC.NOTIFY.ESDC@servicecanada.gc.ca
Display name: NOTIFICATION GC / ACCESSIBLE / GC NOTIFY (EDSC/ESDC)

**Comment by schancase** [comment]
AT has put in a request for an auto-reply. We are waiting on this.

**Comment by connorscarolyns** [comment]
The request was complete; an auto-reply goes out to users who submit feedback, but we are unable to add an inbox notification to let us know when a message is received. We'll need to add a workflow reminder to check this box regularly.
